From e2e5b5c41a1464789d54f3936eca28c27bcbe1e6 Mon Sep 17 00:00:00 2001
From: Hao Li <hao.l.li@intel.com>
Date: Thu, 25 Jan 2018 08:15:45 -0500
Subject: [PATCH 0513/1041] VHM/VBS-K: only notify target vcpu for ioreq
 completion

VBS-K CoE should not set "req->client = 0" since it might cause
issue when workload is heavy.

Change-Id: If7391a68dfbda0d4d347588f33f8958cbe057183
Signed-off-by: Jason Chen CJ <jason.cj.chen@intel.com>
Signed-off-by: Hao Li <hao.l.li@intel.com>
---
 drivers/vbs/vbs_rng.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/vbs/vbs_rng.c b/drivers/vbs/vbs_rng.c
index f5860bab7bab..f928bd7d854d 100644
--- a/drivers/vbs/vbs_rng.c
+++ b/drivers/vbs/vbs_rng.c
@@ -369,7 +369,6 @@ static int handle_kick(int client_id, int req_cnt)
 				 req->reqs.pio_request.size,
 				 req->reqs.pio_request.value);
 			req->processed = REQ_STATE_SUCCESS;
-			req->client = 0;
 			cwp_ioreq_complete_request(client->vhm_client_id, i);
 		}
 	}
-- 
2.17.1

